### ISSUE TITLE
[Bug]: run milvus-backup in k8s pod, retrurn "failed to connect \"Addr\": \"milvus:tcp:\" "

### ISSUE THREAD

**Issue by jackiedlh** [zilliztech/milvus-backup#197]
### Current Behavior

When I run milvus-backup command on k8s pod, the target milvus address is wrong.

in backup.yaml:

milvus:
  address: milvus
  port: 19530
  authorizationEnabled: true
  # tls mode values [0, 1, 2]
  # 0 is close, 1 is one-way authentication, 2 is two-way authentication.
  tlsMode: 0
  user: ""
  password: ""

in log:

[2023/09/08 07:35:33.943 +00:00] [INFO] [logutil/logutil.go:165] ["Log directory"] [configDir=]
[2023/09/08 07:35:33.943 +00:00] [INFO] [logutil/logutil.go:166] ["Set log file to "] [path=logs/backup.log]
[2023/09/08 07:35:33.943 +00:00] [INFO] [core/backup_impl_create_backup.go:25] ["receive CreateBackupRequest"] [requestId=4b980b8b-4e1a-11ee-8398-82b39bdfe663] [backupName=backup] [collections="[]"] [databaseCollections=] [async=false]
[2023/09/08 07:35:34.168 +00:00] [WARN] [grpclog/grpclog.go:46] ["[core][Channel #1 SubChannel #2] grpc: addrConn.createTransport failed to connect to {\n  \"Addr\": \"milvus:tcp:\",\n  \"ServerName\": \"milvus:tcp:\",\n  \"Attributes\": null,\n  \"BalancerAttributes\": null,\n  \"Type\": 0,\n  \"Metadata\": null\n}. Err: connection error: desc = \"transport: Error while dialing dial tcp: address milvus:tcp:: too many colons in address\""]

### Expected Behavior

expect connect to:

milvus:19530

### Steps To Reproduce

```markdown
1 copy milvus-backup tool to k8s milvus proxy pod
2 copy config files to milvus pod
3 run command:  ./milvus-backup create -n backup
```

### Environment

```markdown
aws EKS
```

**Comment by wayblink** [comment]
address should be a ip or a domain.

**Comment by Archalbc** [comment]
> address should be a ip or a domain.

"milvus" is correct in a Kubernetes context, it will resolve to the milvus-proxy pod, actually this look like a bug.

**Comment by nightmorning0** [comment]
It is definitely a bug. Add PORT# at the tail of the address. Some thing like this:
```
milvus:
address: milvus:19530#
port: 19530
authorizationEnabled: false
```
It will comment `:tcp:` from `milvus:tcp::` in your error as a tricky solution.

**Comment by adeelamin15** [comment]
Yup it is definitely a bug. For me workaround is to set the port using `MILVUS_PORT` environment variable. Apparently the bug is causing garbage value in port field which can be overridden using environment variable.

**Comment by cin-eren** [comment]
When you deploy on K8s pod, it is recommended to set MinIO and Milvus configuration via environment variable, example:
```yaml
MILVUS_ADDRESS: milvus
MILVUS_PORT: 19530
MINIO_BUCKET_NAME: s3_bucket
MINIO_ROOT_PATH: ""
MINIO_BACKUP_BUCKET_NAME: s3_backup_bucket
MINIO_BACKUP_ROOT_PATH: backup
MINIO_BACKUP_USE_SSL: true
MINIO_BACKUP_USE_IAM: true
```
Some configuration must be specified in `backup.yaml` like `minio.storageType`, `minio.backupStorageType`, you can consider to use Configmap for them
